.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "multisense_lib/sensor_api"]
+	path = multisense_lib/sensor_api
+	url = git@github.com:carnegierobotics/LibMultiSense.git
```

Comment:
I think this should be the https address instead of the ssh address

Reply:
Sure can do